### ISSUE TITLE
RabbitMQ input duplicating messages

### ISSUE THREAD

**Issue by oksuz** [logstash-plugins/logstash-input-rabbitmq#56]
Hi I tested rabbitmq input before install to production and I've seen a problem. I publish message on rabbitmq management console to queue, and logstash repeating message 3 times. 

Here is my config:

```
input {
    rabbitmq {
        host => "localhost"
        port => 5672
        user => "guest"
        password => "guest"
        queue => "log"
        key => "log"
        threads => 1
        durable => true
    }
}

output {
    stdout { codec => rubydebug }
    elasticsearch { hosts => ["localhost:9200"] }
}
```

output:
![screenshot-4](https://cloud.githubusercontent.com/assets/1628550/11747159/508d4972-a02a-11e5-804e-f8af3065fdd8.png)

Is it a bug ? How can I resolve ?

logstash version: 2.1
rabbitmq version : 3.2.4
os: Ubuntu-server 14.04 64bit

**Comment by oksuz** [comment]
Hi again, I resolved this problem installing 3.1.1 version of plugin with following commands;

```
sudo bin/plugin uninstall logstash-input-rabbitmq
sudo bin/plugin install --version 3.1.1 logstash-input-rabbitmq
```

It seems 3.1.2 bug. Btw I found 3.1.1 version on rubygems. I didn't see in tags. Maybe you forget tagging this verison.
https://rubygems.org/gems/logstash-input-rabbitmq/versions/3.1.1

**Comment by andrewvc** [comment]
Weird, but I'm glad its resolved! Thanks for reporting this @oksuz !
